cmd/notation/cert/list.go

```diff
@@ -22,6 +22,20 @@ func certListCommand(opts *certListOpts) *cobra.Command {
 		Use:     "list",
 		Aliases: []string{"ls"},
 		Short:   "List certificates in the trust store.",
+		Long: `List certificates in trust store
+
+Example - List all certificate files stored in the trust store
+  notation cert ls
+
+Example - List all certificate files of trust store "acme-rockets"
+  notation cert ls --store "acme-rockets"
```

Comment:
same as above

Reply:
@yizha1 For example description, we need to use standard spelling instead of the short alias. This will help users understand what the following CLI commands meaning.